JavaScript/packages/recognizers-date-time/src/dateTime/baseDate.ts

```diff
@@ -35,11 +35,13 @@ export class BaseDateExtractor implements IDateTimeExtractor {
         this.config = config;
     }
 
-    extract(source: string, refDate: Date): Array<ExtractResult> {
-        if (!refDate) refDate = new Date();
+    extract(source: string, refDate: Date): ExtractResult[] {
+        if (!refDate) {
+refDate = new Date();
```

Comment:
It seems the issue @STayinloves mentioned still happens and the auto-fix breaks indentation.